bitnami/cilium/templates/agent/hubble-peers-metrics-service.yaml

```diff
@@ -3,25 +3,25 @@ Copyright Broadcom, Inc. All Rights Reserved.
 SPDX-License-Identifier: APACHE-2.0
 */}}
 
-{{- if .Values.agent.hubbleMetrics.enabled }}
+{{- if .Values.hubble.peers.metrics.enabled }}
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ printf "%s-hubble-metrics" (include "cilium.agent.fullname" .) }}
+  name: {{ printf "%s-hubble-peers-metrics" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
   namespace: {{ include "common.names.namespace" . | quote }}
   labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
     app.kubernetes.io/part-of: cilium
-    app.kubernetes.io/component: hubble
+    app.kubernetes.io/component: hubble-peers
     prometheus.io/scrape: "true"
-  {{- if or .Values.agent.hubbleMetrics.service.annotations .Values.commonAnnotations }}
-  {{- $annotations := include "common.tplvalues.merge" (dict "values" (list .Values.agent.hubbleMetrics.service.annotations .Values.commonAnnotations) "context" .) }}
+  {{- if or .Values.hubble.peers.metrics.service.annotations .Values.commonAnnotations }}
```

Comment:
I see that in the values.yaml we are using {{ }} references, could you move them here? You can see an example in Dremio

Reply:
addressed at https://github.com/bitnami/charts/pull/27526/commits/6e1f6e34bf685be14e71fc180dee89c349f180ab